packages/venia-concept/venia-upward.yml

```diff
@@ -21,6 +24,31 @@ response:
       use: appShell
   default: bundles
 
+imageOpt:
+  inline:
+    status: 301
+    body:
+      inline: ''
+    headers:
+      inline:
+        Location:
+          engine: mustache
+          provide:
+            width: $match.$1
+            path: $match.$2
+          template:
+            when:
+              - matches: env.USE_FASTLY
+                pattern: '[^0]'
+                use:
+                  inline: '/{{path}}?auto=webp&format=pjpg&'
```

Comment:
Do we want to use webp? 

Firefox 65 just added support and Safari doesn't support it at all. There is a polyfill we could use.